.gitignore

```diff
@@ -130,4 +130,8 @@ dmypy.json
 
 demo*/
 .vscode/
-notes.md
\ No newline at end of file
+notes.md
+
+cache/
+index.html
```

Comment:
Unfortunately we can't .gitignore index.html and prompt.txt because Google Cloud Run doesn't upload files which are matched by .gitignore. At least that's my experience - I can't find a reference to this behaviour!

Reply:
https://cloud.google.com/sdk/gcloud/reference/topic/gcloudignore mentions

>  The default content of the generated `.gcloudignore` file, which can be overriden with `--ignore-file`, is as follows:
> 
> ```
>  .gcloudignore
>  .git
>  .gitignore
>  ```
> ...
> In order to ignore files specified in the gitignore file, there is a special comment syntax:
> 
> `#!include:.gitignore`

So you are right

Reply:
Thinking out loud.. given the fact `nldb` is intended to be installed as a package/dependency, it should not contain this `.gitignore` 🤔